### DISCUSSION TITLE
自定义窗口标题还能填其他配置吗

### DISCUSSION THREAD

**Discussion by Ad-closeNN** [Meloong-Git/PCL#3340] [提问] [answered]
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/31a3e83d-601c-4807-a673-192bb7c37ac7)
除了这几个变量，还有什么标记可以设置的吗，没有就关了吧(

**Comment by allMagicNB** [comment]
没有了吧，这是启动器替换的吧

**Reply by Ad-closeNN** [reply]
有什么方法是可以显示服务器什么的，比如在服务器的时候显示`zxdnb on Minecraft服务器`，在单人游戏的时候显示`zxdnb on 单人游戏`的吗

**Reply by MoYuan-CN** [reply, marked as answer]
这不是启动器可以做到的内容，你应当去寻找是否有可用的 Mod
原版的窗口信息支持在后面区分：`Minecraft 1.20.1 - 单人游戏` `Minecraft 1.20.1 - 第三方服务器`

**Reply by Ad-closeNN** [reply]
行吧，关了吧
